definitions.yml

```diff
@@ -312,11 +312,14 @@ definitions:
     required:
       - it-IT
       - en-EN
+      - de-DE
```

Comment:
Since the DE locale is not mandatory even on the IO app, what do you think about to removing it from the required fields?